client/packages/common/src/ui/components/inputs/DatePickers/DateTimePickerInput/DateTimePickerInput.tsx

```diff
@@ -126,7 +130,7 @@ export const DateTimePickerInput: FC<
       minDate={minDate}
       maxDate={maxDate}
       {...props}
-      value={DateUtils.getDateOrNull(props.value)}
+      value={DateUtils.getDateOrNull(props.value, undefined, dateParseOptions)}
```

Comment:
Strictly speaking, you probably don't need this one, as the `value` should always be a Date object (not a string) -- it gets converted on input (onBlur), or is already a Date if using the Picker.

Reply:
yes, good point. I'll just remove the `getDateOrNull` then and see if that causes any problems..